### PR TITLE
Update release.py and service-status.py

### PR DESCRIPTION
**Pull request by willkg**
Picks up these changes:

```
d19f45b Extract bug numbers from commits and list them when using make-bug (#35)
2af6518 Update github actions workflow
76121fb Add test scaffolding
f351b04 Update linting and reformatting to use ruff
850fab2 Print url to tag information GitHub (#45)
8d0d645 fix ruff S113 Probable use of requests call without timeout
f6090eb Fix service-status.py so it works without requests (#42)
```

### CODE REVIEW

File: bin/service-status.py [hunk 1]
```diff
@@ -83,20 +82,17 @@ def get_config():
...
-        print(url)
-        print(f"{resp.status_code}, {resp.content}")
-        raise Exception("Bad return code")
+    """Fetch data from a url
+
+    This raises URLError on HTTP request errors. It also raises JSONDecode
+    errors if it's not valid JSON.
+
+    """
+    fp = urlopen(url, timeout=5)
```
Comment (relud) [inline bin/service-status.py:91]: this needs to suppress S310 or perhaps in pyproject.toml
```suggestion
    fp = urlopen(url, timeout=5)  # noqa: S310
```

**Review by relud — APPROVED** [review]
r+wc

**Comment by willkg** [comment]
^^^ I just fixed the S310 issue. I also wrote up https://github.com/willkg/socorro-release/issues/50 so we don't keep hitting these.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**